### PR TITLE
[WIP]Ukrainian translation

### PR DESCRIPTION
**Pull request by Prosta4okua**

### CODE REVIEW

File: android/assets/jsons/Translations/Other.json [hunk 1]
```diff
@@ -83,6 +86,7 @@
 		Simplified_Chinese:"回合"
 		Spanish:"Turno"
 		Czech:"Tah"
+		Ukrainian:""
```
Comment (lishaoxia1985) [inline android/assets/jsons/Translations/Other.json]: you have not translated it.

Comment (Prosta4okua) [reply android/assets/jsons/Translations/Other.json]: I know, so I write WIP in the pull request name

**Comment by yairm210** [comment]
IF you want to keep this as a WIP, then please resolve the conflics this has with the other translations =)

**Comment by yairm210** [comment]
The longer this keeps being a WIP, the more conflicts you'll need to resolve, that's why I reccomend you to merge small amounts every time =)

**Pull request merged**